StardewArchipelago/GameModifications/Buildings/BuildingMenuArchipelago.cs

```diff
@@ -41,6 +42,25 @@ protected BuildingMenuArchipelago(IModHelper modHelper, ArchipelagoClient archip
 
         public abstract List<BluePrint> GetAvailableBlueprints();
 
+        public Dictionary<string, List<string>> BlueprintDict()
+        {
+            var blueprintDict = new Dictionary<string, List<string>>();

```

Comment:
This is marked as resolved, but not done...

Reply:
For me I'm not sure why a BlueprintData class is necessary, because the game already has one, its just overly thorough.  Just pull that, only use what is necessary, and its naming should be obvious.  I'm shifting to this list since you want one, but I will leave this open even if I fix it just so you can look at it.